README.md

```diff
@@ -2,16 +2,24 @@
     <a href="https://sentry.io" target="_blank" align="center">
         <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
     </a>
-<br/>
-    <h1>Gradle plugin for Sentry Android</h1>
 </p>
 
+_Bad software is everywhere, and we're tired of it. Sentry is on a mission to help developers write better software faster, so we can get back to enjoying technology. If you want to join us [<kbd>**Check out our open positions**</kbd>](https://sentry.io/careers/?utm_source=github&utm_medium=readme&utm_campaign=sentry-android-gradle-plugin)_
+
+Gradle Plugin for Sentry Android
+===========
+[![build](https://github.com/getsentry/sentry-android-gradle-plugin/workflows/Test%20Publishing%20UUID%20Dry%20Run/badge.svg?branch=main)](https://github.com/getsentry/sentry-android-gradle-plugin/actions?query=branch%3Amain)
+[![build](https://github.com/getsentry/sentry-android-gradle-plugin/workflows/Test%20Publishing%20UUID/badge.svg?branch=main)](https://github.com/getsentry/sentry-android-gradle-plugin/actions?query=branch%3Amain)
+[![build](https://github.com/getsentry/sentry-android-gradle-plugin/workflows/Pre%20Merge%20Checks/badge.svg?branch=main)](https://github.com/getsentry/sentry-android-gradle-plugin/actions?query=branch%3Amain)
+[![line count](https://tokei.rs/b1/github/getsentry/sentry-android-gradle-plugin/)](https://github.com/getsentry/sentry-android-gradle-plugin/tree/main)
```

Comment:
nice badge but does it bring any value as a plugin? this would be interesting on the SDKs though, its not a "SDK size" metrics but also helpful

Reply:
just a flair, happy to get rid of it if you prefer not to have it

Reply:
no strong feelings, it was just curiosity if it'd make sense for all the other SDKs